types/lodash/common/function.d.ts

```diff
@@ -840,7 +840,10 @@ declare module "../index" {
          * @param predicate The predicate to negate.
          * @return Returns the new function.
          */
-        negate<T extends (...args: any[]) => any>(predicate: T): T;
+        negate(predicate: () => boolean): () => boolean;
+        negate<A1>(predicate: (a1: A1) => boolean): (a1: A1) => boolean;
+        negate<A1, A2>(predicate: (a1: A1, a2: A2) => boolean): (a1: A1, a2: A2) => boolean;
+        negate(predicate: (...args: any[]) => any): (...args: any[]) => boolean;
     }
 
     interface LoDashWrapper<TValue> {
```

Comment:
Fix the wrapper overloads, too. You may need to split this into `LoDashImplicitWrapper<Tvalue>` and `LoDashImplicitWrapper<TValue>` since `negate(): this;` probably won't work anymore.